client/my-sites/theme/main.jsx

```diff
@@ -188,10 +192,29 @@ const BannerUpsellTitle = ( {
 	translate,
 	isSiteEligibleForManagedExternalThemes,
 	isMarketplaceThemeSubscribed,
+	isThemeAllowedOnSite,
 } ) => {
 	const bundleSettings = useBundleSettingsByTheme( themeId );
 	const isEnglishLocale = useIsEnglishLocale();
 
+	if ( ! isThemeAllowedOnSite ) {
+		return isEnglishLocale ||
+			i18n.hasTranslation(
+				'Access this theme for FREE with a %(personalPlanName)s, %(premiumPlanName)s, or %(businessPlanName)s plan!'
+			)
+			? translate(
+					'Access this theme for FREE with a %(personalPlanName)s, %(premiumPlanName)s, or %(businessPlanName)s plan!',
+					{
+						args: {
+							personalPlanName: getPlan( PLAN_PERSONAL ).getTitle(),
+							premiumPlanName: getPlan( PLAN_PREMIUM ).getTitle(),
+							businessPlanName: getPlan( PLAN_BUSINESS ).getTitle(),
+						},
+					}
+			  )
+			: translate( 'Access this theme for FREE with a Personal, Premium, or Business plan!' );
```

Comment:
I don't think any language will have either phrase translated, so we just need the parameterised plan names one

Reply:
Not sure if your suggestion will have a negative impact. If we just use translate the parametrized, then some countries might see some new plans names while their still on the previous naming.

Reply:
Doh oh yeah!